### DISCUSSION TITLE
database file becomes 0 bytes?

### DISCUSSION THREAD

**Discussion by antrys** [keepassxreboot/keepassxc#7762] [Q&A]
I'm pretty sure this isn't a bug, but rather my ignorance at play.  I keep having to restore from the auto-backup file because my database file becomes 0-bytes.

I am using Syncthing between my my Linux Mint desktop, windows desktop, and android devices and open the database with Both KeepassXC and KeepassDX.   It seems that after I open it with keepassDX is when the database file becomes 0-bytes.

Is there something I should be doing differently?

**Comment by droidmonkey** [comment]
Are you using keeshare?

**Reply by antrys** [reply]
No, I've never really understood keeshare, or took the time to lean about it, didn't seem like it was for my single-user use-case.
I'm just using my same credentials across all devices. 

I did this for years with nextcloud and it worked, but recently switched to using Syncthing.  (Nextcloud became too big of a maintenance time-sink)

At this moment, I'm assuming it's a Syncthing issue or the way the database gets updated doesn't agree with it.  I've just changed the update method to "temporary file moved into place" in KeepassXC to see if that helps.

**Reply by droidmonkey** [reply]
Well if the zeroing out occurs on mobile then this might be a DX issue.

**Reply by antrys** [reply]
Could be, I've been attempting to replicate it and haven't been able to make it do it yet.  It's that infamous "only happens when you're not looking" algo. lol.
